### PR TITLE
doc: add badges for vscode and chrome extensions

### PR DESCRIPTION
**Pull request by kpanot**
Add badges for vscode and chrome extensions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,11 @@
 ## Description
 
 [![Stable Version](https://img.shields.io/npm/v/@o3r/core?style=for-the-badge)](https://www.npmjs.com/package/@o3r/core)
+[![Chrome Devtool](https://img.shields.io/chrome-web-store/v/aejabgendbpckkdnjaphhlifbhepmbne?style=for-the-badge&label=Chrome%20DevTool&color=%23F2C146)](https://chromewebstore.google.com/detail/otter-devtools/aejabgendbpckkdnjaphhlifbhepmbne)
```
Comment (matthieu-crouzet) [inline README.md:9]: Chrome Extension ?

Comment (matthieu-crouzet) [inline README.md:9]:
```suggestion
[![Chrome Extension](https://img.shields.io/chrome-web-store/v/aejabgendbpckkdnjaphhlifbhepmbne?style=for-the-badge&label=Chrome%20Extension&color=%23F2C146)](https://chromewebstore.google.com/detail/otter-devtools/aejabgendbpckkdnjaphhlifbhepmbne)
```

File: README.md [hunk 2]
```diff
@@ -6,8 +6,11 @@
...
+[![VsCode](https://img.shields.io/visual-studio-marketplace/v/AmadeusITGroup.otter-devtools?style=for-the-badge&label=VS%20Code&color=%2352A6E7)](https://marketplace.visualstudio.com/items?itemName=AmadeusITGroup.otter-devtools)
```
Comment (matthieu-crouzet) [inline README.md:10]:
```suggestion
[![VSCode Extension](https://img.shields.io/visual-studio-marketplace/v/AmadeusITGroup.otter-devtools?style=for-the-badge&label=VSCode%20Extension&color=%2352A6E7)](https://marketplace.visualstudio.com/items?itemName=AmadeusITGroup.otter-devtools)
```

File: README.md
Comment (matthieu-crouzet) [inline README.md:1]: Either we put `DevTool` or `Extension` for both, either we don't put anything

**Review by mrednic-1A — APPROVED** [review]

**Pull request merged**